cmd/release-controller/audit_backend_gcs.go

```diff
@@ -34,7 +34,7 @@ func NewGCSAuditStore(bucket string, prefix, userAgent, serviceAccountPath strin
 		options = append(options, option.WithUserAgent(userAgent))
 	}
 	if len(serviceAccountPath) > 0 {
-		options = append(options, option.WithServiceAccountFile(serviceAccountPath))
```

Comment:
This commit is good, move it to a completely different PR and I'll merge it now (i don't want it cluttering up this PR due to complexity).

Reply:
Ok, https://github.com/openshift/release-controller/pull/165